test/stat/stat_test.go

```diff
@@ -88,6 +89,22 @@ func TestCliStatForConduitNamespace(t *testing.T) {
 			t.Fatal(err.Error())
 		}
 	})
+
+	t.Run("test named deploy query", func(t *testing.T) {
+		err := TestHelper.RetryFor(10*time.Second, func() error {
+			fmt.Println("HEREEEEE")
+			out, err := TestHelper.ConduitRun("stat", "deploy/web", "--all-namespaces")
+			if err == nil {
+				t.Fatalf("Unexpected stat error: %v", err)
+			}
+
+			fmt.Println(out)
```

Comment:
unintentional? some other way to validate `out`?

Reply:
unintentional!